plugins/@grouparoo/mailjet/src/lib/export/destinationMappingOptions.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseTypes,
+  SimpleAppOptions,
+} from "@grouparoo/core";
+import { connect } from "./../connect";
+import { MailjetClient } from "../client";
+
+const importantFields = ["name", "firstname"];
+
+export const destinationMappingOptions: DestinationMappingOptionsMethod =
+  async ({ appOptions }) => {
+    const client = await connect(appOptions);
+    const required = getRequiredFields();
+    const known = await getUserFields(client);
+    return {
+      labels: {
+        property: {
+          singular: "Mailjet Contact Property",
+          plural: "Mailjet Contact Properties",
+        },
+        group: {
+          singular: "Mailjet List",
+          plural: "Mailjet Lists",
+        },
+      },
+      properties: {
+        required: required,
+        known: known,
+        allowOptionalFromProperties: false,
+      },
+    };
+  };
+
+const mapTypesFromMailjetToGrouparoo = (fieldKey, mailjetType) => {
+  switch (fieldKey) {
+    case "phone":
+      return "phoneNumber";
+  }
+  const map = {
+    str: "string",
+    int: "integer",
+    datetime: "date",
+    bool: "boolean",
+    float: "float",
+  };
```

Comment:
Do you happen to know what other types Mailjet has?  It might be nice to list them out here so know we don't support them